src/main/java/cubicchunks/worldgen/dependency/RegionDependency.java

```diff
@@ -36,40 +36,24 @@
 public class RegionDependency implements Dependency {
 	
 	private GeneratorStage targetStage;
-	
-	private Set<Long> requiredCubes;
-	
 	private Set<Requirement> requirements;
-
-	private int xLow;
-	private int xHigh;
-	private int yLow;
-	private int yHigh;
-	private int zLow;
-	private int zHigh;
 	
-	
-	public RegionDependency(GeneratorStage targetStage, int radius) {
+	public RegionDependency(Cube cube, GeneratorStage targetStage, int radius) {
 
 		this.targetStage = targetStage;
-		
-		this.xLow = -radius;
-		this.xHigh = radius;
-		this.yLow = -radius;
-		this.yHigh = radius;
-		this.zLow = -radius;
-		this.zHigh = radius;
-	}
-	
-	public RegionDependency(Cube cube, GeneratorStage stage, int xLow, int xHigh, int yLow, int yHigh, int zLow, int zHigh) {
```

Comment:
One instance for all cubes is definitely better. I just want to be able to specify region that isn't a cube, which is what this kind of constructor would be useful for.